app.js

```diff
@@ -1,35 +1,31 @@
-var express = require('express');
-var path = require('path');
-var favicon = require('serve-favicon');
-var logger = require('morgan');
-var cookieParser = require('cookie-parser');
-var session = require('express-session');
-var bodyParser = require('body-parser');
-var passport = require('passport');
+import path from 'path';
+import express from 'express';
+import logger from 'morgan';
+import cookieParser from 'cookie-parser';
+import session from 'express-session';
+import bodyParser from 'body-parser';
+import passport from 'passport';
+import routes from 'routes/';
+import account from 'routes/account';
 
-var routes = require('routes/');
-var account = require('routes/account');
-var sessinCredential = require('common/makeCredential')('session');
-
-
-var app = express();
+const sessinCredential = process.env.KINDLIZED_SESSION;
+const app = express();
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).